src/accessibility-statement.md

```diff
@@ -101,11 +101,11 @@ The [GOV.UK Frontend documentation website](http://frontend.design-system.servic
 
 DAC tested the [Technical Documentation Template](https://tdt-documentation.london.cloudapps.digital/), which is the template for the GOV.UK Frontend documentation website.
 
-To learn more, read the [Technical Documentation Template accessibility statement](https://tdt-documentation.london.cloudapps.digital/accessibility/).
+To learn more, read the [Accessibility statement for Technical Documentation Template and documentation](https://github.com/alphagov/tdt-documentation/blob/main/source/accessibility/index.html.md.erb).
 
 ### Frontend codebase
 
-The GOV.UK Frontend codebase is not a website but a [code repository](https://github.com/alphagov/govuk-frontend) and [npm package](https://www.npmjs.com/package/govuk-frontend).
+The GOV.UK Frontend codebase is not a website but a [code repository on GitHub](https://github.com/alphagov/govuk-frontend) and [npm (Node Package Manager) software package](https://www.npmjs.com/package/govuk-frontend).
```

Comment:
@owenatgov can we describe npm like this, including acronym explanation?

Reply:
So I think that way to describe the thing is fine _but_ I'm a little wary that it doesn't accurately describe where it's going. AI worry a layman may think that it's 'npm, the software package' rather than 'the govuk frontend sofware package, on npm', if you see what I mean. I've made a light suggestion, feel free to adjust as necessary.

```suggestion
The GOV.UK Frontend codebase is not a website but a [code repository on GitHub](https://github.com/alphagov/govuk-frontend) and [a software package on npm (Node Package Manager)](https://www.npmjs.com/package/govuk-frontend).
```